Ports/Android/src/com/codename1/impl/android/InPlaceEditView.java

```diff
@@ -1377,26 +1377,28 @@ public void run() {
     }
 
     private static void setEditMode(final boolean resize) {
-        resizeMode = resize;
-
+        resizeMode = true;

```

Comment:
Won't this break `setEditMode(false)`?

It looks like changes to this method would significantly impact backward compatibility.